scripts/main.py

```diff
@@ -44,21 +44,50 @@ def print_to_console(
             max_typing_speed = max_typing_speed * 0.95
     print()
 
+def attempt_to_fix_json_by_finding_outermost_brackets(json_string):
+    # Attempt to fix JSON by finding the outermost brackets
+    # This is a hacky solution to fix JSON that is missing the outermost brackets
+    # This is a common issue with the OpenAI API
+    # This is not a perfect solution, but it works for the most common cases
+    # it removes unnecessary text around the JSON
+    if cfg.speak_mode:
+        speak.say_text("I have received an invalid JSON response from the OpenAI API. Trying to fix it now.")
+    print_to_console("Attempting to fix JSON by finding outermost brackets\n", Fore.RED,"")
+    try:
+        # Find the first bracket
+        first_bracket_index = json_string.index("{")
+        # Find the last bracket
+        last_bracket_index = json_string.rindex("}")
+        # Slice the string to get the JSON
+        json_string = json_string[first_bracket_index:last_bracket_index + 1]
```

Comment:
Provide confirmation to the console that json fix worked
`print("Apparently I managed to fix it.")`